### DISCUSSION TITLE
Possibility to configure update command

### DISCUSSION THREAD

**Discussion by ernierasta** [bkbilly/lnxlink#91] [Ideas]
Hi! 
Thank you for this amazing project! It very useful.

In my environment I need to update lnxlink by running:

`pipx upgrade lnxlink`

Do you think it will make sense to make update command configurable in config.yaml?

I have no problem to workaround it (by creating "fake pip" :-)), but may be useful also to others.

**Comment by bkbilly** [comment]
I've made some changes which uses the python environment of pipx to run pip and update lnxlink, so it is not needed any configuration from the user side.
This will only work on the next version of the app.

I haven't used pipx on my system, so I didn't notice this issue.
Thank you for letting me know!

**Comment by ernierasta** [comment]
Amazing, thank you!
